lib/generators/audited/templates/install.rb

```diff
@@ -1,30 +1,25 @@
 class <%= migration_class_name %> < <%= migration_parent %>
-  def self.up
-    create_table :audits, :force => true do |t|
-      t.column :auditable_id, :integer
-      t.column :auditable_type, :string
-      t.column :associated_id, :integer
-      t.column :associated_type, :string
-      t.column :user_id, :<%= options[:audited_user_id_column_type] %>
-      t.column :user_type, :string
-      t.column :username, :string
-      t.column :action, :string
-      t.column :audited_changes, :<%= options[:audited_changes_column_type] %>
-      t.column :version, :integer, :default => 0
-      t.column :comment, :string
-      t.column :remote_address, :string
-      t.column :request_uuid, :string
-      t.column :created_at, :datetime
-    end
+  def change
+    create_table :audits do |t|
+      t.belongs_to :auditable, polymorphic: true, index: false
+      t.index [:auditable_type, :auditable_id, :version],
```

Comment:
I think, all separate index definitions should be defined at the bottom (after all column definitions) of a method, as always done in rails migrations

Reply:
And please preserve an old name

Reply:
Do you prefer `auditable_index` ?

Reply:
If you rename this index, this possibly will break upgrading by `rails generate audited:upgrade`